src/components/ReCaptcha/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import scriptLoader from 'react-async-script-loader';
+import withReduxForm from '../../libs/hocs/withReduxForm';
+import ReCaptchaWrapper from './wrapper';
+
+function ReCaptcha(props) {
+    const { lang, ...rest } = props;
+
+    const getUrl = () => {
+        if (lang) {
+            return `https://www.google.com/recaptcha/api.js?render=explicit&hl=${lang}`;
+        }
```

Comment:
put this in other file(get-url.js) in order to test it and then below call it as getUrl(lang)